addons/l10n_ec/data/l10n_latam_identification_type_data.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+	<!-- DATA ON MODULE l10n_latam_base 
+	id						name	sequence	is_vat
+	l10n_latam_base.it_vat	VAT			80		TRUE    #RUC
+	l10n_latam_base.it_pass	Passport	90				#Pasaporte
+	l10n_latam_base.it_fid	Foreign ID	100				#Identificacion del Exterior
+	-->
+	<data noupdate="1">
```

Comment:
Why noupdate?  (it is not meant for the user to play around with those)

Reply:
Removed noupdated